src/main/java/frc/robot/systems/BallHandlingFSM.java

```diff
@@ -62,6 +66,7 @@ public FSMState getCurrentState() {
 	public void reset() {
 		currentState = FSMState.IDLE;
 
+		pushCommandTime = -1;
```

Comment:
magic num? why `-1`?

Reply:
We use pushCommandTime == -1 as an indicator that the solenoid is currently retracted.